Packs/KnowBe4_KMSAT/Integrations/KnowBe4KMSATEventCollector/KnowBe4KMSATEventCollector_description.md

```diff
@@ -0,0 +1,8 @@
+## BaseIntegration Help
+
+Markdown file for integration configuration  help snippet. In this file add:
```

Comment:
lets make sure we dont leave this looking like the example. Be sure to include a warning about rate limits